eoscostarica/eos-rate
# Fix/profile data

### Complete UI task and add several objects validation

#295 Format Votes on BP profile Page
#297 Add links to BPs social media
#294 Add link to BP website
#296 Display BP average rating and number of Votes
#277 Redirect in redux (Query mutation)
#309 Wrong Text in Breadcrumb Navigation

### Steps to test

1. go to EOS RATE
2. click and see all blocks producers list
3. choose one block producer
4. look at all BP profile data (now you can click on the website link and social options links)

Optional Test:
1. use search component (it should be working as expected)
2. type an URL manually and try to access to the profile page of a no existing block producer (e.g. `/block-producers/testnamebp`)

### Checklist
